source/_patterns/01-atoms/button/demo/buttons.twig

```diff
@@ -1,23 +1,33 @@
 {% embed '@atoms/grid/_grid-1-up.twig' with { container: true } %}
   {% block column_1 %}
+    <h1 class="text-center">Buttons</h1>
+    {% include '@molecules/card/_card.twig' with button_helper %}
 
     <h1>Button colors</h1>
     <div class="mb-3">
-      {% for color in colors %}
-        {% include '@atoms/button/_button.twig' with color %}
+      {% for name, rgb in scssColors %}
+        {% include '@atoms/button/_button.twig' with {
+          button_color: name,
+          button_text: name|capitalize,
+        } %}
       {% endfor %}
     </div>
-    <h1>Button tags</h1>
+
+    <h1>Button outline</h1>
     <div class="mb-3">
-      {% for element in elements %}
-        {% include '@atoms/button/_button.twig' with element %}
+      {% for name, rgb in scssColors %}
+        {% include '@atoms/button/_button-outline.twig' with {
+          button_color: name,
+          button_text: name|capitalize,
+        } %}
       {% endfor %}
     </div>
 
-    <h1>Button outline</h1>
+    <h1>Button elements</h1>
     <div class="mb-3">
-      {% for outline in outlines %}
-        {% include '@atoms/button/_button.twig' with outline %}
+      {% for element in elements %}
```

Comment:
This really generic global(?) variable? Where does this come from? And can we change it to something with less of a potential for naming collision?